src/parse.coffee

```diff
@@ -33,8 +33,7 @@ parse = (source, callback) ->
       err = new Error(err.message)
 
     if apiElements
-      apiElements = apiElements.toRefract()
-      apiElements.content = apiElements.content.concat(annotations)
+      apiElements.content.unshift(warning) if warning
```

Comment:
You sent me this link which does the same - https://github.com/apiaryio/fury-adapter-swagger/blob/master/src/parser.js#L1309, so I continued with the practice. I noticed according to docs `apiElements.unshift(...)` should be enough, but IIRC it didn't work (or maybe I've done it incorrectly).